### ISSUE TITLE
Exclude social media links flag ignored?

### ISSUE THREAD

**Issue by LiKuhlmann** [unclecode/crawl4ai#353]
Hi, 
thanks a lot for this project! 
Could it be that the exclude_social_media_links flag has been removed/ignored?
It seems to not work for me, the result still contains social media links

**Comment by unclecode** [comment]
@LiKuhlmann Thanks for trying Crawl4ai. First, upgrade to version 0.4.23; it won't be an issue. You will find it in the CrawlRunConfig. Besides that, you can still pass all these parameters as normal keyword arguments. It supports both versions, and in 0.4.22, some of these links were missed. That's why I guess you are facing this question. So please upgrade and let me know if you encounter any issues.

**Comment by LiKuhlmann** [comment]
Thank you very much for the response @unclecode!
I also realised earlier, that it was probably just missing in the CrawlRunConfig.
I updated to version 0.4.23 and when passing the exclude_social_media_links=True flag like this:
```
result = await crawler.arun(
      url=url,
      exclude_social_media_links=True,
      cache_mode=CacheMode.DISABLED
)
```
It is still returning social media links in the response. E.g. from this website: https://www.airbus.com/en
it returns: https://www.facebook.com/airbus in the result.links
But maybe I have misunderstood this feature and it does return the link in the result, but no further content from it
